frontends/qt/main.cpp

```diff
@@ -453,6 +454,8 @@ int main(int argc, char *argv[])
     QWebChannel channel;
     channel.registerObject("backend", webClass);
     view->page()->setWebChannel(&channel);
+    view->settings()->setAttribute(QWebEngineSettings::JavascriptCanAccessClipboard, true);
+    view->settings()->setAttribute(QWebEngineSettings::JavascriptCanPaste, true);
```

Comment:
Why do we need this?

Reply:
Without it, the `navigator.clipboard` API calls fail on Qt